style/wishlist.css

```diff
@@ -0,0 +1,29 @@
+.card__image > img {
+    width: 30rem;
+    height: 28rem;
+    object-fit: cover;
+}
+.card__buttons {
+    justify-content: center;
+}
+.card__buttons .btn {
+    width: 100%;
+}
+
+.main__container {
+    
```

Comment:
remove extra space from here

Reply:
Applied Prettier code formatting to removed unwanted space.